### PR TITLE
Improve docs for contribution

### PR DESCRIPTION
**Pull request by darkwing**
Now that I've finally got this working, I think this is a good listing of steps to get developers going.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+The development workflow for this project is in flux but this document describes the contribution workflow at this point.
+
+# Setting up your development environment
+
+1. Fork the YouShouldUse repository:  https://github.com/mdn/YouShouldUse
+2. Create a heroku account and deploy YouShouldUse to your account.  A helpful guide can be found here:  https://devcenter.heroku.com/articles/getting-started-with-nodejs
+3. Within your GitHub Account Settings, go to "Personal Access Tokens" and create a token
+4. From the terminal, add your newly created GitHub personal access token to the Heroku config:  heroku config:set OAUTH_TOKEN=#######
+5. Create a test repository which you'll send invalid CSS/Stylus/Sass files to.  Within that repository:
```
Comment (openjck) [inline CONTRIBUTING.md]: Sass isn't supported just yet.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+	1. Navigate to *Settings* > *Webhooks and Services*
+	2. Click *Add webhook*
+	3. Enter `http://{your-heroku-address}.herokuapp.com/hook` as the Payload URL. Leave all
+	   other settings at their default values.
+	4. Click *Add webhook*
```
Comment (openjck) [inline CONTRIBUTING.md]: I notice the same installation instructions are published in `README.md`. How about moving the installation instructions to a separate file called `docs/installation.md` and referring to that document from both this document and the README? Something like this...

> For installation instructions, see `docs/installation.md`

That way, we could reduce the risk that different copies of the installation instructions would get out-of-sync with each other.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+When you commit changes to your test repository, your hook should recognize the changes and execute YouShouldUse on your test repository code!
```
Comment (openjck) [inline CONTRIBUTING.md]: Awesome! Maybe we can also add a note about where to look for comments. It's not totally obvious at the moment.

File: CONTRIBUTING.md (hunk 2 above)
Comment (darkwing) [inline CONTRIBUTING.md]: I disagree, tbh.  I think the docs should have as little redirection as possible.  If anything, we should combine the two docs.

**Pull request merged**